docs/enterprise-setup/implementation-guide.md

```diff
@@ -9,12 +9,11 @@ import TabItem from '@theme/TabItem';
 
 [Airbyte Self-Managed Enterprise](./README.md) is in an early access stage for select priority users. Once you [are qualified for a Self-Managed Enterprise license key](https://airbyte.com/company/talk-to-sales), you can deploy Airbyte with the following instructions.
 
-Airbyte Self-Managed Enterprise must be deployed using Kubernetes. This is to enable Airbyte's best performance and scale. The core components \(api server, scheduler, etc\) run as deployments while the scheduler launches connector-related pods on different nodes.
+Airbyte Self-Managed Enterprise must be deployed using Kubernetes. This is to enable Airbyte's best performance and scale. The core Airbyte components (`server`, `webapp`, `workload-launcher`) run as deployments. The `workload-launcher` is responsible for managing connector-related pods (`check`, `discover`, `read`, `write`, `orchestrator`).
```

Comment:
Would it make sense to expand on the first sentence and hint at more specific requirements listed later? Something like "Airbyte Self-Managed Enterprise must be deployed on a Kubernetes cluster managed with Helm. This is to enable Airbyte's best performance and scale. We support enterprise deployments on AWS, GCP, or Azure using services outlined in this document."